Sources/Tracking/CustomerIO.swift

```diff
@@ -360,7 +360,7 @@ public class CustomerIO: CustomerIOInstance {
         implementation?.screen(name: name, data: data)
     }
 
-    public func automaticScreenView(
+    internal func automaticScreenView(
```

Comment:
is this going to be a breaking change since making public functions internal now?

Reply:
This function is undocumented and was not meant for customers to use. The function currently being marked as `public` is a bug IMO. So, no. I do not plan on introducing a breaking change.